WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/SubFilterViewModel.kt

```diff
@@ -137,18 +137,14 @@ class SubFilterViewModel @Inject constructor(
 
     fun changeSubfiltersVisibility(show: Boolean) {
         val isCurrentSubfilterTracked = getCurrentSubfilterValue().isTrackedItem
-        val isSubfilterListTrackerRunning = readerTracker.isRunning(ReaderTrackerType.SUBFILTERED_LIST)
```

Comment:
I've moved the `isRunning` condition into `start/stop` method in ReaderTracker. So calling start/stop multiple times won't have any sideeffects and we don't need to check `isRunning` on all the calling sites.